UI/window-basic-main-outputs.cpp

```diff
@@ -19,7 +19,7 @@ static void OBSStreamStarting(void *data, calldata_t *params)
 
 	output->delayActive = true;
 	QMetaObject::invokeMethod(output->main,
-			"StreamDelayStarting", Q_ARG(int, sec));
+							  "StreamDelayStarting", Q_ARG(int, sec));
```

Comment:
Style:  This seems to be an unnecessary whitespace change.  Please revert it.

This file seems full of changes like this.  Maybe a find-replace gone bad?